src/Microsoft.OData.ModelBuilder/ODataConventionModelBuilderExtensions.cs

```diff
@@ -26,7 +26,9 @@ public static ODataConventionModelBuilder EnableLowerCamelCase(this ODataConvent
             return builder.EnableLowerCamelCase(
                 NameResolverOptions.ProcessReflectedPropertyNames |
                 NameResolverOptions.ProcessDataMemberAttributePropertyNames |
-                NameResolverOptions.ProcessExplicitPropertyNames);
+                NameResolverOptions.ProcessExplicitPropertyNames | 
+                NameResolverOptions.ProcessEnumMemberNames
```

Comment:
This should not be added to the default options, there will be existing callers who could break because the generated model is now different.

Reply:
I was contemplating myself before making this change. Since this was reported as an issue and the expectation that the enum values are also camel cased, I added it to this extension method.

But I agree with your comment that this could break the existing applications using the default extension method as the model will now suddenly be camel cased and the caller might not be expecting it.